src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
+}
```

Comment:
Java Enum is a Java type-class used to define collections of constants, so you need to name each of them according to style guide. Check [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values)

```suggestion
public enum Colors {
    BLACK,
    WHITE,
    RED,
    BLUE
}
```